R/bambu-extendAnnotations-utilityExtend.R

```diff
@@ -44,17 +44,16 @@ isore.extendAnnotations <- function(combinedTranscripts, annotationGrangesList,
   }
 }
 
-#' filter transcripts by read counts and NDR
+#' filter transcripts by read counts
 #' @noRd
-filterTranscripts <- function(combinedTranscripts, min.sampleNumber, max.txNDR){
+filterTranscripts <- function(combinedTranscripts, min.sampleNumber){
   if (nrow(combinedTranscripts) > 0){
     #filter by read counts
     filterSet <- combinedTranscripts$NSampleReadCount >= min.sampleNumber & (
-          combinedTranscripts$NSampleReadProp >= min.sampleNumber) & (
-            combinedTranscripts$NSampleGeneScore >= min.sampleNumber) & (
-              combinedTranscripts$NSampleTxScore >= min.sampleNumber)
-    combinedTranscripts = combinedTranscripts[filterSet,]
+        combinedTranscripts$NSampleTxScore >= min.sampleNumber) & (
+        combinedTranscripts$NSampleReadProp >= min.sampleNumber)
   }  
+  combinedTranscripts = combinedTranscripts[filterSet,]
```

Comment:
that line might break if `(nrow(combinedTranscripts) <= 0)` as filterSet is not defined outside the if?

Reply:
dealt with in f60dd70a3c2427ec8bcc64e0bf048d047b31c496

Reply:
```suggestion

```

Reply:
If filterSet is NULL then combinedTranscripts will be a table with 0 rows similar to if everything is filtered out. This means the same condition can be used to check if its empty because there were no transcripts or if there were no valid transcripts in isore.extendAnnotations. If combinedTranscripts could sometimes == NULL then 
> if (nrow(NULL)<0) will cause an error.